react/features/video-menu/components/native/VolumeSlider.tsx

```diff
@@ -138,16 +161,18 @@ class VolumeSlider extends PureComponent<IProps, IState> {
  * @param {Object} ownProps - The own props of the component.
  * @returns {IProps}
  */
-function mapStateToProps(state: IReduxState, ownProps: any) {
+function mapStateToProps(state: IReduxState, ownProps: IProps) {
     const { participantID } = ownProps;
-    const { participantsVolume } = state['features/participants-pane'];
+    const { participantsVolume } = state['features/filmstrip'];
     const { startSilent } = state['features/base/config'];
+    const tracks = getTrackState(state);
 
     return {
         _startSilent: Boolean(startSilent),
-        _volume: participantID && participantsVolume[participantID]
+        _track: getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.AUDIO, participantID),
+        _volume: participantID ? participantsVolume[participantID] : undefined
```

Comment:
why did you rewrite this prop? the previous form provides the same result.

Reply:
Reverted.